ga/src/main/scala/cilib/ga/RandomSearch.scala

```diff
@@ -0,0 +1,28 @@
+package cilib
+package ga
+
+import cilib.Lenses._
+import scalaz.std.list._
+import scalaz.syntax.traverse._
+
+// Author: Kyle Erwin
+object RandomSearch {
+
+    def ga[S](parentSelection: List[Individual] => RVar[List[Individual]], deviation: Double): List[Individual] => Individual => Step[Double, List[Individual]] =
```

Comment:
We should rather consider making the this a derivative of the standard GA? you would simply pass in an identity function for any kind of crossover?

Reply:
Okay cool. I can do that